bootstrap/src/main/java/io/jenkins/jenkinsfile/runner/bootstrap/commands/JenkinsLauncherOptions.java

```diff
@@ -68,6 +68,11 @@
             description = "Port for exposing the web server and Jenkins Web UI from Jenkinsfile Runner. Disabled by default")
     public Integer httpPort;
 
+    @CheckForNull
+    @CommandLine.Option(names = "--httpPath",
```

Comment:
FYI for `jenkins.war` the analogous option is

```
   --prefix                 = add this prefix to all URLs (eg http://localhost:8080/prefix/resource). Default is none
```

though that name might be confusingly generic-sounding for something that only affects HTTP service which you would normally not use in JFR to begin with.

Reply:
I prefer the current option. Maybe `--httpPrefix` but no strong opinion

Reply:
I'll leave it as it is then.